### ISSUE TITLE
gpmol dependency

### ISSUE THREAD

**Issue by DavidWalz** [aspuru-guzik-group/gpHSP#1]
@beangoben
First of all, thanks for providing model and data in form of a well formatted notebook. It's exciting to see that this good practice is spreading more and more across different fields!

However, I'm currently missing the gpmol dependency, which seems to contain the most integral parts of the analysis. Is it a group-internal python package? If so, would you consider open sourcing it or removing this dependency? With gpmol missing the analysis is neither reproducible, nor is gpHSP open-source as claimed in the paper.

**Comment by beangoben** [comment]
Hi @DavidWalz,
We wrapping work on gpmol and should have open-sourced soon (end of June), the delay has been mostly life-stuff...we definitely want it to be shared, improved and reproduced.

**Comment by DavidWalz** [comment]
Great!
Looking at the paper, it does shed a bad light on the reviewers who apparently didn't even bother checking out the code.
I do appreciate your effort , and am looking to testing out your model.

**Comment by gbrault** [comment]
Thanks for releasing this work open source, and of course waiting for the gpmol dependency!

**Comment by beangoben** [comment]
definitely, will have a usable dependency in 1 week approx, along with an improved model (multioutput gp, deep gp).
Thanks for the patience.

**Comment by anyistudio** [comment]
@beangoben Thanks for sharing all the data. Just wondering if you still have the plan to share the gpHSP source code?

**Comment by rodrija** [comment]
Please inform when the gpmol code would be released. It is unfortunate that nothing has happened since June last year. 
I found your paper very useful and would like to test your code for solvent screening.

**Comment by ignaczgerg** [comment]
Hey! Is there any news regarding to make gpmol open source?

**Comment by beangoben** [comment]
Thanks for your interest, I'll aim to put the code up next Friday (~21)!

**Comment by ignaczgerg** [comment]
Many thanks! We are really looking forward to it :)

**Comment by sptiwari** [comment]
Any news about the gpmol?

**Comment by sptiwari** [comment]
Hi beangoben, We are really looking forward to run the gpHSP. Could you please upload whatever you have about the gpmol. It need not be perfect. We will be more than happy with just a running version of gpHSP.

Thanks a lot.

**Comment by ignaczgerg** [comment]
Hi! Is there any news regarding the release of gpmol? Thanks.

**Comment by xuzhang5788** [comment]
@beangoben 
When will you release gpmol?

**Comment by thava** [comment]
@beangoben  Any insights into usage of gpmol or alternatives would be highly appreciated.

**Comment by sptiwari** [comment]
@beangoben Any update? We have been waiting for so long!

**Comment by hududed** [comment]
This is frustrating when we can't follow-up on a publication due to technicality - I didn't know academia is not fond of sharing knowledge. Could it be that gpmol is being withheld for IP reasons ?

**Comment by madi20210904** [comment]
Is there any news regarding the release of gpmol? Thanks.

**Comment by jmohin** [comment]
Still waiting on release of GPmol -- coming up on 4 years now.  Would love to reproduce these results and do some extensions. Some [other group's papers](https://github.com/Tabor-Research-Group/redox_mol_screening) are claiming it as a dependency, which seems to indicate to me this library is shareable.

**Comment by beangoben** [comment]
Hi, the most recent edits (march '22) (notebooks 0 to 3) and gphsp.py cover all code required to generate features/train models and predict without requiring gpmol.

The closest thing to gpmol is https://github.com/aspuru-guzik-group/dionysus/.

**Comment by jmohin** [comment]
Thanks! I'll play with the notebooks.

**Comment by jmohin** [comment]
The notebooks fail to import gpHSP due to a dependency on gpFlow within the gphsp.py file:
ModuleNotFoundError: No module named 'gpflow'

**Comment by jmohin** [comment]
nevermind- was able to satisfy that dependency
